### PR TITLE
[miio] Update channels only if they are linked

### PR DESCRIPTION
**Pull request by marcelrv**
With introduction of miot devices with enormous amount of channels
refreshes can become slow. With this change only relevant updates are
done.

Signed-off-by: Marcel Verpaalen <marcel@verpaalen.com>

### CODE REVIEW

**Comment by marcelrv** [comment]
@phuongpham this should also address bit of your slowness

File: bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoBasicHandler.java [hunk 1]
```diff
@@ -318,6 +318,11 @@ private boolean refreshProperties(MiIoBasicDevice device) {
         int maxProperties = device.getDevice().getMaxProperties();
         JsonArray getPropString = new JsonArray();
         for (MiIoBasicChannel miChannel : refreshList) {
+            if (!isLinked(miChannel.getChannel())) {
+                logger.info("Skip refresh of channel {} for {} as it is not linked", miChannel.getChannel(),
+                        getThing().getUID());
```
Comment (cpmeister) [inline bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoBasicHandler.java:323]: This isn't something that the user is required to be notified about.

```suggestion
                logger.debug("Skip refresh of channel {} for {} as it is not linked", miChannel.getChannel(),
                        getThing().getUID());
```

**Review by cpmeister — CHANGES_REQUESTED** [review]

**Comment by phuongpham** [comment]
> @phuongpham this should also address bit of your slowness

Will the update also be rollout to  2.5.11.xx?

File: bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoBasicHandler.java (hunk 1 above)
Comment (marcelrv) [reply bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoBasicHandler.java:323]: Yes, should have changed that after testing

Signed-off-by: Marcel Verpaalen marcel@verpaalen.com

**Review by cpmeister — APPROVED** [review]

**Comment by phuongpham** [comment]
@marcelrv  it will only be added to version 3?

**Pull request merged**